src/Microsoft.ML.Transforms/MissingValueDroppingTransformer.cs

```diff
@@ -28,6 +28,31 @@
 
 namespace Microsoft.ML.Transforms
 {
+    internal sealed class MissingValueDroppingEstimator : TrivialEstimator<MissingValueDroppingTransformer>
```

Comment:
Is this useful in other cases? Do we need to add this to the catalog and make it public?

Reply:
I don't think it is - seems to be something very specific to Ngram extraction.

---
In reply to: [370393234](https://github.com/dotnet/machinelearning/pull/4696#discussion_r370393234) [](ancestors = 370393234)